RendererCommon/Texture.cs

```diff
@@ -0,0 +1,83 @@
+﻿using ImageMagick;
```

Comment:
Please remove comments, we'll add summary tags later for documentation.

Reply:
Done 👍